src/Models/ObjectMap/Entities/EntityGubbins.php

```diff
@@ -142,15 +150,17 @@ public function setFields(array $fields)
     }
 
     /**
-     * @return AssociationStubBase[]
+     * @return AlgoWeb\PODataLaravel\Models\ObjectMap\Entities\Associations\AssociationStubMonomorphic[]|AlgoWeb\PODataLaravel\Models\ObjectMap\Entities\Associations\AssociationStubPolymorphic[]|array
      */
     public function getStubs()
     {
         return $this->stubs;
     }
 
     /**
-     * @param AssociationStubBase[] $stubs
+     * @param AlgoWeb\PODataLaravel\Models\ObjectMap\Entities\Associations\AssociationStubMonomorphic[]|AlgoWeb\PODataLaravel\Models\ObjectMap\Entities\Associations\AssociationStubPolymorphic[]|DateTime[]|array $stubs
+     *
```

Comment:
this is automated based on the data that has been passed, some how in tests Stubs is being set to a DateTime[]. is that deliberate?